test/visibility/import/enablesUnqualified/multipleUnqualified/useAndImport-useExcept1b.chpl

```diff
@@ -0,0 +1,27 @@
+// Tests behavior when a module is both imported and used from different paths
+// when the use has an except list, ensuring that we can still access the
+// remaining symbols and utilize qualified access.
+module A {
+  var x: int = 4;
+  proc foo() {
+    writeln("In A.foo()");
+  }
+}
+module B {
+  public import A.{foo};
+}
+
+module C {
+  public use A except x;
+}
+
+module D {
+  use B, C;
+
+  proc main() {
+    // writeln(x); // Won't work
+    writeln(A.x);
```

Comment:
This is subtle... Does `public use A except x` bring `A` to the scope but not `x` and we can use that `A` for qualified access? My intuition is that this line should also not work

Reply:
I think I got my answer reading the test with `public use A except *`